holidays/countries/british_virgin_islands.py

```diff
@@ -0,0 +1,187 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import JUN, OCT, DEC
+from holidays.groups import ChristianHolidays, InternationalHolidays, StaticHolidays
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SAT_TO_PREV_FRI,
+    SUN_TO_NEXT_MON,
+    MON_TO_NEXT_TUE,
+    SAT_SUN_TO_NEXT_MON,
+    SAT_SUN_TO_NEXT_MON_TUE,
+)
+
+
+class BritishVirginIslands(
+    ObservedHolidayBase, ChristianHolidays, InternationalHolidays, StaticHolidays
+):
+    """British Virgin Islands holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_the_British_Virgin_Islands>
+        * [Public Holidays (Amendment) Act, 2021](https://laws.gov.vg/Laws/public-holidays-amendment-act-2021)
+        * <https://www.timeanddate.com/holidays/british-virgin-islands/>
+        * [2015](https://bvi.gov.vg/media-centre/2015-public-holidays)
+        * [2016](https://bvi.gov.vg/media-centre/2016-public-holidays)
+        * [2017](https://bvi.gov.vg/media-centre/2017-public-holidays)
+        * [2018](https://bvi.gov.vg/media-centre/2018-public-holidays)
+        * [2019](https://bvi.gov.vg/media-centre/2019-public-holidays)
+        * [2020](https://bvi.gov.vg/media-centre/2020-public-holidays)
+        * [2021](https://bvi.org.uk/2021-public-holidays/)
+        * [2022](http://www.bvi.gov.vg/media-centre/revised-2022-public-holidays)
+        * [2023](https://bvi.gov.vg/media-centre/updatedofficialholidays20231)
+        * [2024](https://bvi.gov.vg/media-centre/cabinet-approves-2024-public-holidays)
+        * [2025](https://bvi.gov.vg/media-centre/2025-holiday-calendar-flyer-01)
+    """
+
+    country = "VG"
+    default_language = "en_VG"
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    supported_languages = ("en_US", "en_VG")
+    start_year = 1967
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, BritishVirginIslandsStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_TO_PREV_FRI + SUN_TO_NEXT_MON)
```

Comment:
```suggestion
        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
```
General note on the observed holidays changes. Since I don't consider Timeanddate to be a reliable enough source in this matter, I find it possible to simplify the implementation slightly.